go.mod

```diff
@@ -2,12 +2,15 @@ module github.com/celestiaorg/celestia-app/v4
 
 go 1.23.6
 
+// TODO: tag multiplexer as v0.1.0 after merge.
+replace github.com/celestiaorg/celestia-app/multiplexer => ./multiplexer
```

Comment:
@julienrbrt is the idea to pin to a specific version, or just have a local replace in the go.mod?

Pros/cons to both approaches, local pin reduces operational overhead of additional tags needing to be required in order to test changes.

Depending on pin makes it so you don't need to copy source code in the Dockerfile and that it's clear which version of the multiplexer you are using.

I kind of saw the multiplexer as just being a sub-component of the existing celestia-app, and that there wasn't really a need to have it tagged and released independently if there is just a single project using it.

cc @rootulp

Reply:
I thought it would be a separate go module yeah, but if it isn't i can change.

Reply:
we can do local pin for now anyway and do follow up, with the tag 👍

Reply:
Great question. Local pin _seems_ preferable based on the OP so inclined to stick with that if we don't need to release it separately.